vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -800,29 +801,34 @@ void Publisher::UpdateMediaDuration(
                 this,
                 _1,
                 _2,
+                window_id,
                 publisher_key,
                 duration));
 }
 
 void Publisher::OnGetPublisherInfoForUpdateMediaDuration(
     ledger::Result result,
     ledger::PublisherInfoPtr info,
+    const uint64_t window_id,
     const std::string& publisher_key,
```

Comment:
is this used?

Reply:
It's ultimately used like this in the visit logic in `publisher.cc`:

```
    if (window_id > 0) {
      OnPanelPublisherInfo(ledger::Result::LEDGER_OK,
                           std::move(panel_info),
                           window_id,
                           visit_data);
    }
```

My understanding is that since the legacy code isn't using a callback to fetch the panel info, `window_id` is used to ensure that that the returned data ends up being associated with the correct tab.

Reply:
no I mean `publisher_key`

Reply:
Ahh, didn't catch that. Removed.